spec/spec_helper.rb

```diff
@@ -68,4 +68,4 @@ def wasi_debug
   end
 end
 
-at_exit { GC.start(full_mark: true) }
+at_exit { GC.start(full_mark: true) } if ENV["GC_AT_EXIT"] == "1"
```

Comment:
Can you expand on your rationale behind this change?

Reply:
It's really only necessary for valgrind, since Ruby might not free everything at exit otherwise from what I gather. Stole it from nokogiri.